source/Octopus.Tentacle/Diagnostics/Log.cs

```diff
@@ -2,12 +2,13 @@
 using System.Collections.Concurrent;
 using System.Collections.Generic;
 using Octopus.Diagnostics;
+using Octopus.Tentacle.Contracts.Logging;
 using Octopus.Tentacle.Diagnostics.Formatters;
 using Octopus.Tentacle.Util;
 
 namespace Octopus.Tentacle.Diagnostics
 {
-    public abstract class Log : ILog, IDisposable
+    public abstract class Log : ILog, ITaskLog, IDisposable
```

Comment:
Just wondering, does this have to implement both `ILog` and `ITaskLog`?

Reply:
I think this is needed for the tests to compile, but really the Integration tests should just all be using the to be renamed `ITaskLog`